crypto/asn1/a_strex.c

```diff
@@ -200,6 +200,8 @@ static int do_buf(unsigned char *buf, int buflen,
             unsigned char utfbuf[6];
             int utflen;
             utflen = UTF8_putc(utfbuf, sizeof(utfbuf), c);
```

Comment:
Blank line between decls/inits and body.
```suggestion
            int utflen = UTF8_putc(utfbuf, sizeof(utfbuf), c);

```